CONTRIBUTING.md

```diff
@@ -145,7 +145,7 @@ Skip the yarn part and proceed to the Pushing the changes section [here](#-pushi
 
 ## 📌 Pushing your changes
 
-Now you have made the changes , tested them, and built them. so now its time to push them.
+Now you have made the changes , tested them, and built them so now its time to push them.
```

Comment:
Remove space before comma.
Change it to:
Once you made the changes, and tests are successfully passing you can push your changes to remote.

Reply:
Change this sentence to : `Once you made the changes, and tests are successfully passing you can push your changes to remote.`